target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
Here is the new partitioning layout : 

```
Part    Offset          Size            Name
mtd0    0x10000000      0x20000         logic_config
mtd1    0x10020000      0x20000         logic_addr
mtd2    0x10040000      0x20000         bootmonitor
mtd3    0x10080000      0x40000         flash_config
mtd4    0x100c0000      0x40000         preloader
mtd5    0x100e0000      0x1F20000       firmware
```
The preloader contains only an asm jump to the address 100e0040 (firmware plus the uImage header size offset 0x40).